flytekit/core/task.py

```diff
@@ -416,3 +416,35 @@ def wrapper(fn) -> ReferenceTask:
         return ReferenceTask(project, domain, name, version, interface.inputs, interface.outputs)
 
     return wrapper
+
+
+class Echo(PythonTask):
+    _TASK_TYPE = "echo"
+
+    def __init__(self, name: str, inputs: Optional[Dict[str, Type]] = None, **kwargs):
+        """
+        A task that simply echoes the inputs back to the user.
+        The task's inputs and outputs interface are the same.
+        FlytePropeller won't create a pod for this task, it will simply pass the inputs to the outputs.
+        https://github.com/flyteorg/flyte/blob/master/flyteplugins/go/tasks/plugins/testing/echo.go
```

Comment:
As a follow up, I think this should link out to a docs around "How to enable echo tasks". This way a user can look at the docstring and know to contact their platform engineer to enable the feature.

Reply:
Thanks! good call.
- [PR](https://github.com/flyteorg/flytesnacks/pull/1724) for 1
- [PR](https://github.com/flyteorg/flyte/pull/5679) for 2

Reply:
I will create another follow-up PR to address this bad error. 

![image](https://github.com/user-attachments/assets/256992c4-6be1-4912-af12-5acc7fd3d7ba)

Reply:
I just updated the docstring in this PR too